From 1a61fe8ed3b84d7ead94b69eac11dee6f2394b97 Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Tue, 25 Jun 2024 14:49:55 +0200
Subject: [PATCH 1008/1695] core: lib: scmi-server: add log prefix support for
 SCP firmware

Add support for conditional option to enable prefix when logging SCP
firmware traces.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git dfb9bc814e77257dcaf523faf6fb11ab12521308 core: lib: scmi-server: add log prefix support for SCP firmware]
Change-Id: I69032463afc074dc1091a89dd52940f3e19dd30d
Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/389491
---
 core/lib/scmi-server/sub.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/lib/scmi-server/sub.mk b/core/lib/scmi-server/sub.mk
index 695a5419c..9250b606c 100644
--- a/core/lib/scmi-server/sub.mk
+++ b/core/lib/scmi-server/sub.mk
@@ -67,6 +67,7 @@ cppflags-lib-y += -DFWK_LOG_LEVEL=$(CFG_SCPFW_LOG_LEVEL)
 ifneq ($(CFG_SCPFW_LOG_LEVEL),0)
 cppflags-lib-y += -DFMW_LOG_MINIMAL_BANNER=1
 endif
+cppflags-lib-$(CFG_SCP_ENABLE_LOG_PREFIX) += -DFWK_LOG_LEVEL_PREFIX
 
 cflags-lib-y += -Wno-cast-align \
 		-Wno-nonnull-compare \
-- 
2.47.2

